ftw/zipexport/upgrades/20161129110908_add_events_to_zip_creation/registry.zcml

```diff
@@ -0,0 +1,4 @@
+<registry>
```

Comment:
This two interfaces are event marker interfaces, there are not persisted and registered in the plone.app.registry. 

There is no profile change, so there is no upgradestep needed, please remove.